src/Asset/EntrypointLookupInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Symfony\WebpackEncoreBundle\Asset;
+
+
+interface EntrypointLookupInterface
+{
+    public function getJavaScriptFiles(string $entryName): array;
```

Comment:
Now that I see this interface, I think we should "clean it up" a little bit. I think we should create a new exception class - e.g. `EntrypointNotFoundException` and advertise above this method in phpdoc that this exception should be thrown if the entry does not exist.